app-modules/campaign/database/migrations/2023_10_11_204551_create_campaigns_table.php

```diff
@@ -44,7 +44,6 @@ public function up(): void
         Schema::create('campaigns', function (Blueprint $table) {
             $table->uuid('id')->primary();
             $table->foreignUuid('user_id')->constrained('users');
-            $table->foreignUuid('caseload_id')->constrained('caseloads');
```

Comment:
It will be removed in a migration, but we should and also need to remove this here because of the constraint on a table that no longer exists. Existing tenants will have this wiped by the data migration, but new tenants/local environments shouldn't add this column at all (and can't because it would fail).